### ISSUE TITLE
QA

### ISSUE THREAD

**Issue by Whitecathnb** [Whitecathnb/DL#1]
One solution to solve two problems：
First,the softmax function that directly implements the softmax operation based on the mathematical definition may lead to overflow;
Second, cross_ Entropy is implemented according to the definition of cross entropy loss function, and there may also be overflow.
 For details, refer to the question at the bottom of the code:
[QA](https://github.com/Whitecathnb/DL/blob/master/d2l_learning/Softmax_regression/Realization_of_Softmax_regression_from_zero.ipynb
)
